themes/bootstrap3/templates/RecordTab/holdingsils.phtml

```diff
@@ -118,7 +119,9 @@
   <?php endif; ?>
 </table>
 <?php endforeach; ?>
-
+<?php if ($options['itemLimit'] && $paginator = $this->tab->getPaginator()): ?>
+	<?=$this->paginationControl($paginator, 'Sliding', 'Helpers/pagination.phtml', ['page' => $options['page']])?>
```

Comment:
This line should only require two spaces of indentation.

Reply:
Done

Reply:
Thanks!